tests/integration/ceph_upgrade_test.go

```diff
@@ -172,6 +190,11 @@ func (s *UpgradeSuite) TestUpgradeToMaster() {
 	cephfsFilesToRead = append(cephfsFilesToRead, newFile)
 	logger.Infof("Verified upgrade from v1.4 to master")
 
+	created = utils.Retry(48, 5*time.Second, "OBC is created", func() bool {
```

Comment:
If we need to wait for the rgw after upgrade, can we have a separate wait for the rgw update from the OBC retry?

Reply:
Actually, we can get rid of this entirely since the OBC controller runs parallel to the main orchestration. Good thought.